File: hot-framework/src/test/resources/be/solidx/hot/test/nio/http/js/https-client-server-keycert.js
var options = {
	url: 'https://localhost:8443/test',
	ssl: {
		ca:'ca/ca.crt',
		key:"ca/client.key",
		cert:"ca/client.crt",
		passphrase:"client"
	}
}

hprint("Starting...")

client.buildRequest(options).done(function(data) {
	hprint(new java.lang.String(data))
	server.stop()
})

File: hot-framework/src/test/resources/be/solidx/hot/test/nio/python/https-client-server-keycert.py
options = {
	'hostname': 'localhost',
	'port': 8443,
	'path': "/test",
	'ca':'ca/ca.crt',
	'key':"ca/client.key",
	'cert':"ca/client.crt",
	'passphrase':"client",
}

class Body:
	pass
	
def responseHandler(response):

    print "connected"
    body = Body()
    body.content = ""
	
    def dataHandler (chunck):
        body.content += chunck

    def endHandler ():
        print body.content
        print response.statusCode

    response.on("data",dataHandler)
    response.on("end", endHandler)

request = client.request (options , responseHandler)
request.end()